docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature: `840` Add a new notification UI. Backend errors should now display a notification on the upper right corner.
```

Comment:
Shouldn't you also specify difference between warnings and errors here?

Reply:
I am not sure it makes sense from a changelog perspective. Warnings didn't change that much from the previous version. There is just a slight visual change of the UI, but other than that they behave exactly the same way as before. Only the behavior for the backend errors changed.

Reply:
Okay np